app/views/idv/cac/choose_method.html.erb

```diff
@@ -1,24 +1,45 @@
 <%= render 'shared/maintenance_window_alert' do %>
+  <style>
+    .flex-container {
+        display: flex;
+    }
+    @media (max-width: 30em) {
+        .flex-container {
+            flex-direction: column;
+        }
+    }
+  </style>
   <h1 class="h3 mt0 mb1"><%= t('cac_proofing.headings.choose_method') %></h1>
-  <%= t('cac_proofing.info.choose_method') %>
+  <p><%= t('cac_proofing.info.choose_method') %></p>
+  <p><%= t('cac_proofing.info.disclaimer') %></p>
+  <div class="flex-container mt3 mb3">
```

Comment:
I think we should try to avoid BassCSS if we can here, to make our future migration easier, especially when there are some pretty direct alternatives for margins.

```suggestion
  <div class="flex-container margin-top-4 margin-bottom-4">
```

There's also a short-hand if applying the same vertical margin to top and bottom:

```suggestion
  <div class="flex-container margin-y-4">
```

Reply:
Fixed.